packages/core/admin/admin/src/content-manager/components/DynamicTable/CellContent/utils/tests/hasContent.test.js

```diff
@@ -50,6 +50,23 @@ describe('hasContent', () => {
     });
   });
 
+  describe('boolean', () => {
+    it('returns true if enabled', () => {
+      const normalizedContent = hasContent('boolean', true);
+      expect(normalizedContent).toEqual(true);
+    });
+
+    it('returns false if disabled', () => {
+      const normalizedContent = hasContent('boolean', false);
+      expect(normalizedContent).toEqual(true);
```

Comment:
This should be `.toEqual(false);`, shouldn't it?

Reply:
Ah sorry I think I missunderstood what "hasContent" does. But maybe the naming of the test is missleading then?
`it('returns false if disabled', () => {` should be `it('returns true if disabled', () => {`

Reply:
Thanks! :) You are right - the name of the test was misleading. The behavior itself is fine because we want to show `true` or `false` as strings in the CM and only nothing for `null`. Therefore `false` means, the field contains something that we can display.